engine/installation/linux/ubuntulinux.md

```diff
@@ -49,64 +49,90 @@ your `APT` sources to the new Docker repository.
+    ```bash
+    $ sudo apt-get update
+    $ sudo apt-get install apt-transport-https ca-certificates
+    ```
+4.  Add the new `GPG` key.
 
-4. Add the new `GPG` key.
+    ```bash
+    $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+    ```
 
-        $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
-
-5. Open the `/etc/apt/sources.list.d/docker.list` file in your favorite editor.
+5.  Open the `/etc/apt/sources.list.d/docker.list` file in a text editor.
 
     If the file doesn't exist, create it.
 
-6. Remove any existing entries.
-
-7. Add an entry for your Ubuntu operating system.
-
-    The possible entries are:
-
-    - On Ubuntu Precise 12.04 (LTS)
+6.  Remove any existing entries.
 
-            deb https://apt.dockerproject.org/repo ubuntu-precise main
+7.  Add the entry for your Ubuntu operating system.
 
-    - On Ubuntu Trusty 14.04 (LTS)
+    The entry determines where APT will search for packages. The possible entries
+    are:
 
-            deb https://apt.dockerproject.org/repo ubuntu-trusty main
+    | Ubuntu version      | Repository                                                  |
+    | ------------------- | ----------------------------------------------------------- |
+    | Precise 12.04 (LTS) | `deb https://apt.dockerproject.org/repo ubuntu-precise main`|
+    | Trusty 14.04 (LTS)  | `deb https://apt.dockerproject.org/repo ubuntu-trusty main` |
+    | Xenial 16.04 (LTS)  | `deb https://apt.dockerproject.org/repo ubuntu-xenial main` |
 
-    - Ubuntu Xenial 16.04 (LTS)
 
-            deb https://apt.dockerproject.org/repo ubuntu-xenial main
+    >**Note**: Docker does not provide packages for all architectures. Binary artifacts
+    are built nightly, and you can download them from
+    https://master.dockerproject.org. To install docker on a multi-architecture
+    system, add an `[arch=...]` clause to the entry. Refer to
+    [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
+    for details.
 
-    > **Note**: Docker does not provide packages for all architectures. You can find
-	> nightly built binaries in https://master.dockerproject.org. To install docker on
-    > a multi-architecture system, add an `[arch=...]` clause to the entry. Refer to the
-    > [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
-    > for details.
+8.  Save and close the `/etc/apt/sources.list.d/docker.list` file.
 
-8. Save and close the `/etc/apt/sources.list.d/docker.list` file.
+9.  Update the `APT` package index.
 
-9. Update the `APT` package index.
-
-        $ sudo apt-get update
+    ```bash
+    $ sudo apt-get update
+    ```
 
 10. Purge the old repo if it exists.
 
-        $ sudo apt-get purge lxc-docker
+    ```bash
+    $ sudo apt-get purge lxc-docker
```

Comment:
We should check if this is still an issue, `lxc-docker` is really ancient now, so not very likely that people run into this